api/base/types.go

```diff
@@ -41,13 +41,14 @@ type ModelStatus struct {
 
 // Machine holds information about a machine in a juju model.
 type Machine struct {
-	Id         string
-	InstanceId string
-	HasVote    bool
-	WantsVote  bool
-	Status     string
-	Message    string
-	Hardware   *instance.HardwareCharacteristics
+	Id          string
+	InstanceId  string
+	DisplayName string
```

Comment:
I can't see the CLI code which uses this - I guess it's still in progress?

Reply:
This struct is used by `api/common/modelstatus.go`, `api/common/modelmanager.go`, et al